frontend/ui/elements/screensaver_menu.lua

```diff
@@ -165,7 +173,7 @@ return {
                         text = _("Cycle through images in order"),
                         help_text = _("When enabled, all images (up to 128) will be displayed at least once on the sleep screen in sequence before repeating the cycle."),
                         enabled_func = function()
-                            return G_reader_settings:readSetting("screensaver_type") == "random_image"
+                            return mayUseRandomImageFolder()
```

Comment:
Shouldn't that just be `enabled_func = mayUseRandomImageFolder`?

On a separate note, allowRandomImageFolder or canUseRandomImageFolder seems like it would fit in better with the rest of the code.